pkg/trace/api/api.go

```diff
@@ -481,6 +489,9 @@ func (r *HTTPReceiver) handleTraces(v Version, w http.ResponseWriter, req *http.
 		ClientComputedTopLevel: req.Header.Get(headerComputedTopLevel) != "",
 		ClientComputedStats:    req.Header.Get(headerComputedStats) != "",
 	}
+
+	payload.ClientDroppedP0s = extractDroppedP0sHeaders(req, ts)
```

Comment:
Can we put this into the `payload` definition above? I don't think it needs to be here...